### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (11 - Clear Words

### ISSUE THREAD

**Issue by jspellman** [w3c/wcag3#264]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

[Note: see screenshots from Word document]
 
The first line of the guideline is  “Use clear words” which is an imperative.  As with 7.1 this is non-normative so it is ok as an imperative – but I would suggest we move away from them even in opening lines like this.   In WCAG 2 they were used to open up a whole section.  Having an imperative for each provision – I think is confusing and I would advise against it – and moving toward the Outcome orientation approach cited in the introduction to WCAG 3.0  
The next part is an exception.   This is very confusing on a number of fronts. 
•	First – the exception comes before the Objective – which is the normative part of the provision. Having an exception before the requirement is confusing.   Perhaps this is just an editorial oversight.
•	Second the wording of the exception is a problem – but I’ll discuss this below after going through the rest of the parts of this provision.
Next comes the outcome-oriented requirement “Common clear words”. However, although it is labeled in the markup as the objective – and is in the same location as the objective in 7.1 – it looks more like a title than an outcome.    I would expect an outcome to be worded more like “content is available in clear common words”. 
Another problem with this provision is that it is not objective unless the words ‘clear’ and ‘common’ are clearly and unambiguously defined.   That is:
•	Exactly what words fall into the category of “clear and common”
•	Is there a list that I can choose from?
•	Do all my words have to be clear and common- or just some?
•	What about proper names (my name or the name of my city or state or county or the name of my school)?  Some are common names, but most are not.
•	How would I go about determining if my site conformed in a way that all testers would come to the same conclusion that I passed or failed. 
•	I cannot see how this could be tested – or that anyone could objectively and reliably determine that something had passed or failed – and I could not find any guidance on this in the supporting documentation. 
•	Presumably this accessibility guideline would/should meet its own requirements for simple language.  (Or is this a technical document that would be subject to the exception?) 
The exception for technical documents
This provision has an exception as follows: 
Exception: Technical documents that provide a clear language summary
The exception would be normative – so also need to be clear and objective.  This would first require a definition of a technical document.   Is this a technical document?   Clearly a paper on quantum physics would be.  How about an advanced cookbook?   Is the word “sauteed” a clear and common word?  How about knitting?  Can one have a site dedicated to it that does only uses plain language and no jargon? Or wood working?   Or fishing?   Or any site on any topic that is not intended for young audiences?   Almost any topic is going to have jargon and words that are not understood by someone unfamiliar with that topic.  Where is the line?   How does one know if they have crossed it (to the good or bad side) (to the conforming or non-conforming side). 
For technical documents a different rule is provided (labelled as an extension).  But it is unclear what constitutes a summary.   For the site on quantum or string theory – would the following summary qualify?   “This is a site about what everything is made of.”   For the WCAG 3.0  - would the following be sufficient summary “This is a document about how to make web pages easier to use by people with disabilities.”?   That is clear and accurate but would not seem to be very complete for someone with a disability that pulled up the guidelines to find out what they contained.  But that would meet the letter of the exception.  And everything after that is opinion – and if we asked everyone on the committee what constituted an adequate summary, I would wager we would get as many different opinions as there are members – or more. 
As I mentioned – this is really hard to do. 
As an exercise – I suggest that each person in the working group look at their own website or that of their organization and apply this provision to the first 5 pages.   Several of us tried this in WCAG 2 and it was a real eye opener.   I thought it would be very hard.   And I completely underestimated the task.  And never did find a means to figure out what which words I could use.  I even did a study of reading level measures and discovered all the limitations and ways they fail to address this. 
I have no idea how to meet this provision.  I would love to see the results of the application by the working group members to their sites.    That is a good test for this provision – and for all new provisions.   (In WCAG 2.0 we had a rule that we needed 3 implementations (sites) that met all of the guidelines before we could advance them.  It was a lot of work, but it was highly educational and led to changes in the guidelines.)  
Use of the technical provision to dodge this provision 
Note that without a clear definition of “technical document” this provision could be easily met by simply putting a single sentence summary of each page, at the bottom of the page.  
Can one say that anything that somebody can’t understand is too technical for them?
